src/solvers/refinement/string_constraint_generator.h

```diff
@@ -115,10 +115,6 @@ class string_constraint_generatort final
 
   symbol_generatort fresh_symbol;
 
-  symbol_exprt fresh_univ_index(const irep_idt &prefix, const typet &type);
-
-  symbol_exprt fresh_exist_index(const irep_idt &prefix, const typet &type);
-
```

Comment:
From a formal point of view, having these being handled the same is a little concerning.  I'm guessing this is just a naming artefact rather than something deeper.